budgetApp1/budgetApp1/Constants.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  Constants.swift
+//  budgetApp1
+//
+//  Created by Lehlohonolo Nkadimeng on 2024/04/22.
+//
+
+import Foundation
+
+// MARK: - URL constants
+struct URLConstants {
+    static let accessToken = "?access_token=t_Ikm7Q2U94be-lmSn4blgh1sjIEf1dMF7Tv8jguvpU"
+    static let baseURL = "https://api.ynab.com/v1/"
+    static let budgetURL = baseURL + "budgets" + accessToken
+    static let accountsURL = baseURL + "budgets/2b0c25e3-0d5e-411d-a019-2522faa22c2b/accounts" + accessToken
+    static let transactionsURL = baseURL + "budgets/2b0c25e3-0d5e-411d-a019-2522faa22c2b/accounts/0515719a-f641-422b-8341-7073d5358dbe/transactions" + accessToken
+}
+
+// MARK: - nib identifiers
+struct NibIdentifiers {
+    static let AccountViewCellIdentifier = "AccountViewCell"
```

Comment:
these variables can still follow camel case. 
accountViewCellIdentifier, transactionViewCellIdentifier

Reply:
this was resolved but not changed?
Please do not mark the conversation s resolved until the change is made or you provide feedback as to why you are going to keep it?

Reply:
Done.